### PR TITLE
Docs for sql params

### PR DESCRIPTION
**Pull request by mazameli**
Let me know if you have anything to add or change, @salsakran.

### CODE REVIEW

**Comment by camsaul** [comment]
`[ci skip]` only works if you put it in the git commit message, not in the PR title 💔

**Comment by mazameli** [comment]
Got it. Thanks, Cam!

**Comment by mazameli** [comment]
Want to look at this and make edits or give it a +1, @salsakran ?

File: docs/users-guide/12-sql-parameters.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+This example defines a variable called `cat`, allowing you to dynamically change the `WHERE` clause in this query:
+```
+SELECT count(*)
+FROM products
+WHERE category = {{cat}}
+```
+
+#### The `Field filter` variable type
+Giving a variable the `Field filter` type allows you to connect SQL cards to dashboard filter widgets. A field filter variable inserts SQL similar to that generated by the GUI query builder when adding filters on existing columns. This is important because it lets you do things like insert dynamic date range filters into your native query. When adding a field filter, you should link that variable to a specific column. Dimensions should be used inside of a `WHERE` clause.
```
Comment (salsakran) [inline docs/users-guide/12-sql-parameters.md]: maybe "useful" instead of "important"?

Comment (salsakran) [inline docs/users-guide/12-sql-parameters.md]: There's a stray "dimensions" in the last sentence.

File: docs/users-guide/12-sql-parameters.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+[[WHERE category = {{cat}}]]
+```
+
+To use multiple optional clauses you must include at least one regular `WHERE` clause followed by optional clauses, each starting with `AND`.
+
+Example:
+```
+SELECT count(*)
+FROM products
+WHERE 1=1
```
Comment (salsakran) [inline docs/users-guide/12-sql-parameters.md]: can we use `WHERE True` instead of `WHERE 1=1`?

**Comment by salsakran** [comment]
Two nits picked and one dangling "dimension". 

I'm a 👍 once the dimension is replaced with `Field Based Filter Clause Insertion Point`.

**Pull request merged**